response_operations_ui/__init__.py

```diff
@@ -45,20 +44,19 @@ def create_app(config_name=None):
     def user_loader(user_id):
         return User(user_id)
 
-    if app.config['SESSION_TYPE'] == 'redis':
-        cf = ONSCloudFoundry()
-        # If deploying in cloudfoundry set config to use cf redis instance
-        if cf:
-            redis_service_name = app.config['REDIS_SERVICE']
+    if cf.detected:
+        with app.app_context():
```

Comment:
What's this? Feels strange that we'd have to use a context here

Reply:
ah so it works locally but in the cf it kept throwing `RuntimeError: Working outside of application context.`
To overcome this I had to use `app_context`

Reply:
:+1: Yeah this is a sensible place to use it

Potentially you could just pass the app object into `ONSCloudfoundry(...)` instead but this solution works fine